source/templates.tex

```diff
@@ -3129,23 +3135,21 @@
 
 \pnum
 \indextext{specialization!class template partial}%
-A partial specialization of a class template provides an alternative definition
+A partial specialization of a template provides an alternative definition
 of the template that is used instead of the primary definition when the
 arguments in a specialization match those given in the partial
-specialization\iref{temp.class.spec.match}.
-A declaration of the primary template shall precede any specialization of
+specialization\iref{temp.spec.partial.match}.
+A declaration of the primary template shall precede any partial specialization of
```

Comment:
The addition of "partial" here is wrong (this also applies to explicit specializations).

Reply:
Thanks, reverted.